lib/assesments/assessment_2/amna/screens/login/widget/_fi_as2_amna_button.dart

```diff
@@ -0,0 +1,32 @@
+part of '../fi_as2_amna_login.dart';
+
+class _Button extends StatelessWidget {
+  final int i;
+  final String value;
+  const _Button({required this.i, required this.value});
+
+  @override
+  Widget build(BuildContext context) {
+    return InkWell(
+      onTap: () {},
+      child: Container(
+        width: 330.0,
+        decoration: BoxDecoration(
+            color: i == 0 ? Colors.transparent : FiAs2AmnaColors.primaryColor,
```

Comment:
instead of taking i as an int take it a bool, since there is no particular reason for having an int if ultimately it's being converted to a bool, 
or just take a widget as backgroundColor and add the color depending on that,
both of these approaches are cleaner than using an int

Reply:
I need i as an int for this part. 
`  const _Button(
              i: 0,
              value: "Login",
            ),
            const _Button(
              i: 1,
              value: "Signup",
            )`